### ISSUE TITLE
Incorrect formatting of MODS informations

### ISSUE THREAD

**Issue by KnutLohse** [slub/dfg-viewer#84]
Given following snippet:
`<mods:titleInfo>`
`  <mods:title>Türkische Post</mods:title>`
`  <mods:subTitle>Tageszeitung für den Nahen Osten</mods:subTitle>`
`  <mods:partName>Halbmonatliche Wirtschaftsausgabe</mods:partName>`
`</mods:titleInfo>`

Informations are concatenated to _"Türkische Post Halbmonatliche Wirtschaftsausgabe"_
(see http://dfg-viewer.de/show/?tx_dlf%5bpage%5d=1&tx_dlf%5bid%5d=http%3A%2F%2Fzefys.staatsbibliothek-berlin.de%2Fgoobi%2FZDB24378574_1940_year_mets.xml&tx_dlf%5bdouble%5d=0&cHash=980fd7890b6800ce8bcaf0fab83b3e42&no_cache=1)

That is incorrect:
1. multiple informations have to be divided (description char) as: _"Türkische Post / Halbmonatliche Wirtschaftsausgabe"_
2. what's about the sub title?

**Comment by albig** [comment]
1. I don't get the point. Divided by which description char? Do you mean "Türkische Post **in** Halbmonatliche Wirtschaftsausgabe"?
2. Question to @sebastian-meyer: We currently don't output the mods:subTitle. Shouldn't we do that?

**Comment by KnutLohse** [comment]
Lieber Herr Bigga,

habe die Frage an meine Kollegin Fr. Federbusch weitergegeben (ich nix Bibliothekar).   ;-)
Vermutlich soll ein Slash Verwendung finden. Habe aber auch angefragt, wie die Ausgabe formatiert werden soll, wenn _alle drei_ Werte angezeigt würden.
Bitte etwas Geduld…

Mit freundlichen Grüßen
Knut Lohse
--
__________________________________________________________________
Knut Lohse
Informations- und Datenmanagement, Referat 3.3, Innovation/Entwicklung
Staatsbibliothek zu Berlin – Preußischer Kulturbesitz

Tel.:   +49 30 266-43 23 31

Von: Alexander Bigga [mailto:notifications@github.com]
Gesendet: Mittwoch, 18. Oktober 2017 18:06
An: slub/dfg-viewer <dfg-viewer@noreply.github.com>
Cc: Lohse, Knut <Knut.Lohse@sbb.spk-berlin.de>; Author <author@noreply.github.com>
Betreff: Re: [slub/dfg-viewer] Incorrect formatting of MODS informations (#84)


  1.  I don't get the point. Divided by which description char? Do you mean "Türkische Post in Halbmonatliche Wirtschaftsausgabe"?
  2.  Question to @sebastian-meyer<https://github.com/sebastian-meyer>: We currently don't output the mods:subTitle. Shouldn't we do that?

**Comment by KnutLohse** [comment]
Hier nun unsere Gedanken zum Thema Anzeige der Daten:

generell: den Zusatz zum Sachtitel (<mods:subTitle>)  müsste man (vermutlich aus Platzgründen) nicht unbedingt mit auswerfen, wenn dann mit Spatium Doppelpunkt Spatium in folgender Form:
_Juristenblatt : Informationen, Meinungen, Analysen_

die Unterreihe (<mods:partName>) (im Beispiel Halbmonatliche Wirtschaftsausgabe) sollte unbedingt zusätzlich angezeigt werden – Schrägstrich ist richtig:
_Türkische Post / Halbmonatliche Wirtschaftsausgabe_

Jetzt ist die Frage, wie bei Angabe der vollen Schönheit angezeigt werden sollte:
_Türkische Post : Tageszeitung für den Nahen Osten / Halbmonatliche Wirtschaftsausgabe_ 
Also: **mods:title : mods:subTitle / mods:partName**

**Comment by sebastian-meyer** [comment]
We omit mods:subTitle just because there is not enough space to display all title variants and the other metadata. The only thing missing here is the separator between mods:title and mods:partName. But that's only a configuration issue. I'll change that asap.

**Comment by sebastian-meyer** [comment]
Die Anzeige sollte *mods:title / mods:partName* lauten - auch für die Jahresdatei, nur dass dort die Angaben aus mods:relatedItem[@type="host"] kommen. Das Nachladen der Überordnung zum Auslesen des Titels ist aus Performance-Gründen im DFG-Viewer nicht möglich.

**Comment by beatrycze-volk** [comment]
@sebastian-meyer is this issue still relevant?

**Comment by sebastian-meyer** [comment]
Yes, I think so. At least the original issue is still valid: Between `mods:title` and `mods:partName` has to be a slash.
